paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/CustomerSheetTest.kt

```diff
@@ -79,26 +85,40 @@ internal class CustomerSheetTest {
             response.testBodyFromFile("payment-methods-create.json")
         }
 
-        networkRule.enqueue(
-            retrieveSetupIntentRequest(),
-            retrieveSetupIntentParams(),
-        ) { response ->
-            response.testBodyFromFile("setup-intent-get.json")
-        }
-
-        networkRule.enqueue(
-            confirmSetupIntentRequest(),
-            confirmSetupIntentParams(),
-        ) { response ->
-            response.testBodyFromFile("setup-intent-confirm.json")
+        when (customerSheetTestType) {
```

Comment:
Can we add this block as a function on `CustomerSheetTestType`? Looks like we have a few of these that are all basically the same.